lib/appoptics_apm/frameworks/sinatra.rb

```diff
@@ -47,26 +48,76 @@ def appoptics_rum_footer
       end
       alias_method :oboe_rum_footer, :appoptics_rum_footer
     end
+
+    module Templates
```

Comment:
I don't understand what this is about at all.

Reply:
Yes, I consolidated all the instrumentation for Sinatra in one file.